### PR TITLE
Fix HAML linting warnings

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Review by kntsoriano — APPROVED** [review]
Thanks!

File: app/views/layouts/mailer.html.haml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 !!!
 %html
   %head
-    %meta{:content => "text/html; charset=utf-8", "http-equiv" => "Content-Type"}/
+    %meta
+      { :content => "text/html; charset=utf-8", "http-equiv" => "Content-Type" }/
```
Comment (iHiD) [inline app/views/layouts/mailer.html.haml]: What was wrong with this one out of interest? Line too long?

Comment (ErikSchierboom) [reply app/views/layouts/mailer.html.haml]: It was the missing spaces after the opening curly brace and before the closing. I've just rechecked and now with the latest Rubocop version, it has changed this line again, but differently.

File: .prettierignore [hunk 2]
```diff
@@ -1 +1,2 @@
 *.rb
+*.haml
```
Comment (ErikSchierboom) [inline .prettierignore:2]: Apparently, prettier formats .haml files too by default, which conflicted with `haml-lint` so I've excluded them from prettier.

File: app/views/layouts/mailer.html.haml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 !!!
 %html
   %head
-    %meta{:content => "text/html; charset=utf-8", "http-equiv" => "Content-Type"}/
+    %meta{ :content => "text/html; charset=utf-8", "http-equiv" => "Content-Type" }/
```
Comment (iHiD) [inline app/views/layouts/mailer.html.haml]:
```suggestion
    %meta{ content: "text/html; charset=utf-8", "http-equiv": "Content-Type" }/
```

**Comment by iHiD** [comment]
Thanks :)

**Pull request merged**